integration-cli/docker_api_containers_test.go

```diff
@@ -552,7 +552,7 @@ func (s *DockerSuite) TestContainerApiCommit(c *check.C) {
 	if err != nil {
 		c.Fatal(err)
 	}
-	if cmd != "{[/bin/sh -c touch /test]}" {
```

Comment:
Or, put another way, when we see lines like: https://github.com/docker/docker/blob/master/api/types/types.go#L105  should replace the ref to runconfig.Config with either an in-line struct or another struct defined within that same file (making it clear that its just for exporting/serialization and not internal), and then that would force us to then write the code to map runconfig.Config to this new struct.  In the end, it seems like nothing in types.go should reference a struct/type defined outside of that file.